crates/re_renderer/src/context.rs

```diff
@@ -106,6 +123,32 @@ impl RenderContext {
     ) -> Self {
         re_tracing::profile_function!();
 
+        let frame_index_for_uncaptured_errors = Arc::new(AtomicU64::new(STARTUP_FRAME_IDX));
+
+        // Make sure to catch all errors, never crash and deduplicate reported errors.
```

Comment:
The oxford comma helps disambiguate ("never deduplicate reported errors")

```suggestion
        // Make sure to catch all errors, never crash, and deduplicate reported errors.
```